packages/api/src/platforms/vtex/resolvers/validateCart.ts

```diff
@@ -128,7 +128,7 @@ const joinItems = (form: OrderForm) => {
       const [item] = items
       const quantity = items.reduce((acc, i) => acc + i.quantity, 0)
       const totalPrice = items.reduce(
-        (acc, i) => acc + i.quantity * i.sellingPrice,
+        (acc, i) => acc + i.priceDefinition.total,
```

Comment:
priceDefinition.total may not exist in some cases like when you have an item unavailable for ffm. 

Example of orderForm with priceDefinition null:
https://pedrocruzio.myvtex.com/api/checkout/pub/orderForm/05a783b41db3433a9287f6c863e79d98

Can you adjust to use this field when it exist otherwise add zero? Only at cases that the item is not actually sellable to the customer this field will be null.